docs/installation.rst

```diff
@@ -31,48 +31,61 @@ Setting up Vagrant
 
      >vagrant ssh
 
-Setting up Treeherder
----------------------
-
 * A python virtual environment will be activated on login, all that is left to do is cd into the project directory:
 
   .. code-block:: bash
 
      (venv)vagrant@precise32:~$ cd treeherder-service
 
+* Build the log parser Cython files, since they are required for both running the tests and a local Treeherder instance
+
+  .. code-block:: bash
+
+     (venv)vagrant@precise32:~/treeherder-service$ python setup.py build_ext --inplace
+
+  NB: If you change something in the treeherder/log_parser folder, remember to repeat this step, otherwise the changes will not take effect.
+
+Running the tests
+-----------------
+
+* The tests can be run without following the local instance steps below.
+
 * You can run the py.test suite with
 
   .. code-block:: bash
 
      (venv)vagrant@precise32:~/treeherder-service$ ./runtests.sh
```

Comment:
I've got a not-yet-pushed-but-blocked-on-this branch which adds that for bug 1091750 :-)